### ISSUE TITLE
Non-ASCII characters in preceding folders

### ISSUE THREAD

**Issue by leorolla** [vlmantova/bookml#18]
If the full path to the current folder contains non-ASCII characters, new folders with corrupted names will be created, some files will be put in subfolders of the corrupted folder, and the HTML will not work.

I have tested this on Ubuntu 24.04

This is a problem of LaTeXML, not of BookML.

Perhaps BookML should issue an error message and exit?

**Comment by xworld21** [comment]
> This is a problem of LaTeXML, not of BookML.

Are you able to produce a minimal example, say in the `/tmp` folder, so I can reproduce it locally?

If this is truly a pure latexml issue, it would be good to report it to them directly. Once I understand the issue, I can figure out if BookML can do something helpful (I am not too optimistic).

**Comment by leorolla** [comment]
cd /tmp/
    mkdir test
    cd test
    mkdir commeça
    cd commeça
    cat > main.tex <<EOF
    \documentclass{article}
    \begin{document}
    Hello World!
    \end{document}
    EOF
    wget https://github.com/vlmantova/bookml/releases/download/v0.9.1/release.zip
    unzip release.zip
    cp bookml/GNUmakefile .
    make
    ls ..
    firefox auxdir/html/main/index.html

There will be two folders above, and too few files in the right place:

    $ls ..
    commeÃ§a  commeça
    $ ls auxdir/html/main/
    imsmanifest.xml  index.html  search_index.json

**Comment by xworld21** [comment]
Indeed, the problem is that the Perl Cwd module returns *bytes* that need to be decoded before use. I'll report this upstream.
